src/st/strategy/api-hooks/class-wpml-pb-api-hooks-strategy.php

```diff
@@ -19,13 +19,13 @@ public function register_strings( $post ) {
 	}
 
 	/**
-	 * @param bool $registered
 	 * @param string|int $post_id
 	 * @param string $content
+	 * @param WPML\PB\Shortcode\StringCleanUp $stringCleanUp
 	 *
 	 * @return bool
 	 */
-	public function register_strings_in_content( $post_id, $content, $do_cleanup ) {}
+	public function register_strings_in_content( $post_id, $content, WPML\PB\Shortcode\StringCleanUp $stringCleanUp ) {}
```

Comment:
@brucepearson, it's a bit strange to see `WPML\PB\Shortcode\StringCleanUp` (shortcode) in the API strategy. An interface would be more suitable (`WPML\PB\StringCleanUp`). Or maybe, if you want to keep it simple, you can just keep the type in the PHPDoc and remove the type hint in the signature.

Reply:
Well, actually this `WPML\PB\Shortcode\StringCleanUp` is not specific to shortcodes because we are passing the strategy as a dependency. So you just need to use a better namespace ( e.g. `WPML\PB\StringCleanUp`).

Reply:
`WPML\PB\Shortcode\StringCleanUp $stringCleanUp` is clean up code specific to shortcodes so I don't want to remove it from the namespace. The API strategy doesn't use it or implement any functionality for `register_strings_in_content`.
So I think it's ok to keep this as it is.

Reply:
OK, you call @brucepearson!